test/core/integration/concrete/lockup-dynamic/create-with-timestamps-ld/createWithTimestampsLD.tree

```diff
@@ -0,0 +1,34 @@
+CreateWithTimestampsLD_Integration_Concrete_Test
```

Comment:
don't agree, it is not scalable

Reply:
> abstract only contains very basic tests

bingo, why duplicate it then?

the "scalable" part was mainly due to this explanation: https://github.com/sablier-labs/v2-core/pull/1083#discussion_r1840863009

Reply:
I thought about it more last night. I have come to realize that I am against this mainly because of BTT. If it were not for BTT, I would have supported this refactor.

To be compatible with BTT, this is how tests should be written:
1. Write a test tree first.
2. Generate test contract using `bulloak scaffold`
3. Remove definitions of modifiers as they are defined in parent contracts but not from the tests
4. Update logic inside the tests

In your child contract of create-with-timestamps, if you use `bulloak scaffold` it wont be able to add all those modifiers since they do not exist in the tree anymore. `bulloak check` is not failing because it does not flag additional modifiers. Therefore, creating a test contract from those trees has is no longer automated now because you would have to add modifier by yourself manually. So if you miss a modifier or two, bulloak check wont detect that because they are no longer a part of the tree structure.

I hope I have made myself clear why I am againt this.